library/core/src/main/java/com/google/android/exoplayer2/source/chunk/ChunkSource.java

```diff
@@ -118,4 +120,11 @@ void getNextChunk(
 
   /** Releases any held resources. */
   void release();
+
+  /**
+   * Returns the PRFT box when available.
+   * @return PRFT box
+   */
+  @Nullable
+  ProducerReferenceTimeBox getProducerReferenceTimeBox();
```

Comment:
Maybe it is better to move it to DefaultDashChunkSource or DashChunkSource?

Reply:
ChunkSource is the interface, and the method is implemented in DefaultDashChunkSource